css/styles.css

```
@@ -128,3 +134,110 @@ ul {
     color: var(--Grey);
     padding-bottom: 20px;  
 }
+
+/* Search module */
+
+.logo {
+    text-align: center;
+    margin: 0 auto;
+    padding-top: 110px;
+}
+
+.logo img {
+    width: 250px;
+}
+
+.search {
+    text-align: center;
+    padding: 12px; 
+}
+
+.search input {
+   border: none;
+   width: 450px;
+}
+
+.search-wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 550px;
+    height: 40px;
+    border-radius: 20px;
+    border: 1px solid var(--Alto);
+    margin: 0 auto;
+}
+
+.search-icon-right {
+    display: flex;
+    align-items: center;
+    padding: 0 10px;
+    gap: 10px;
+}
+
+.search-icon-left {
+    display: flex;
+    align-items: center;
+    padding: 0 10px;
+}
+
+.google-buttons {
+   text-align: center;
+   
+}
+
+.google-buttons__search {
+    background-color: var(--Concrete);
+    color: var(--Grey);
+    border: 1px solid var(--Concrete);
+    border-radius: 5px;
+    font-size: 12px;
+    margin: 12px 6px;
+    padding: 10px 15px;
+    cursor: pointer;
+}
+
+/* Footer */
+
+footer {
```

```suggestion
.footer {
```
We should use classes to modify element(s) appearance